dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/WebContainer.java

```diff
@@ -1018,9 +1018,18 @@ public void run() {
             if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
                 Tr.event(tc, "startModule: " + webModule.getName() + "; " + e);
             }
-            
-            //PI58875
-            this.stopModule(moduleInfo);
+           
+            //issue 25855 - SRVE0164E is "context.root.already.in.use" exception.  The module has not fully started yet
+            //  so we need to skip certain operation (like move the VirtualHost with context root ...in order to not interrupt
+            //  the started app.  Pass in boolean.
+            if (e.getMessage().contains("SRVE0164E")) {
+                this.stopModule(moduleInfo, false);
+            }
```

Comment:
I'm not a fan of checking the message text like this but I'm not familiar enough with how the web container works to suggest another way of doing it.

Reply:
SRVE0164E is the message code which is registered and should never changed in service.